libs/designer/src/lib/ui/panel/nodeDetailsPanel/tabs/createConnectionTab.tsx

```diff
@@ -52,6 +57,8 @@ const CreateConnectionTab = () => {
   const [selectedSubscriptionId, setSelectedSubscriptionId] = useState('');
   const gatewaysQuery = useGateways(selectedSubscriptionId, connector?.id ?? '');
   const availableGateways = useMemo(() => gatewaysQuery.data, [gatewaysQuery]);
+  const gatewayServiceConfigQuery = useGatewayServiceConfig();
+  const gatewayServiceConfig = useMemo(() => gatewayServiceConfigQuery.data, [gatewayServiceConfigQuery]);
```

Comment:
Adjustment from the other hook suggestion

```suggestion
  const gatewayServiceConfig = useGatewayServiceConfig();
```